go.work

```diff
@@ -1,4 +1,4 @@
-go 1.19
+go 1.22
```

Comment:
Can we leave this at 1.21, like what we are doing with the rest, compile using 1.22, set go mod to 1.21 using `go mod tidy -go=1.21 -compat=1.21`